spec/spec_models.cr

```diff
@@ -230,6 +231,22 @@ end
       end
     end
 
+    class ReservedWord{{ adapter_const_suffix }} < Granite::ORM::Base
+      adapter {{ adapter_literal }}
+      table_name "{{ reserved_word_table }}"
+      field all : String
+
+      def self.drop_and_create
+        exec "DROP TABLE IF EXISTS #{quoted_table_name}"
+        exec <<-SQL
+          CREATE TABLE #{quoted_table_name} (
+            id {{ primary_key_sql }},
+            #{quoted_column_name("all")} VARCHAR(255)
+          )
+        SQL
+      end
+    end
```

Comment:
Thank you for following the pattern with your tests in this pull request, it makes it a lot easier to see how the test suite is actually covering all the code.